proto/proto-backplane-grpc/src/main/proto/io/deephaven/proto/console.proto

```diff
@@ -201,9 +201,265 @@ message FetchFigureRequest {
     io.deephaven.proto.backplane.grpc.Ticket console_id = 1;
     reserved 2;
 
+    string figure_name = 3;
```

Comment:
is this what the reserved field was for? if applicable, please add a comment of the intended use and link a github issue (similar to what you did for the table map reference in figure descriptor).

Reply:
reserved was originally for a result ticket, but based on what a ticket means today to deephaven, i dont think we need that any more, will remove.